patches/chromium/export_gin_v8platform_pageallocator_for_usage_outside_of_the_gin.patch

```diff
@@ -0,0 +1,33 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <samuel.r.attard@gmail.com>
+Date: Tue, 3 Nov 2020 16:49:32 -0800
+Subject: export gin::V8Platform::PageAllocator for usage outside of the gin
+ platform
+
```

Comment:
Can you add an explanation here w/ upstream plans?

Reply:
lint failed here :( this needs a description @MarshallOfSound

Reply:
Oops, missed this comment.  How did it go green if lint was failing :/

Reply:
sorry, i meant lint failed to detect the missing description because of the two-line subject